### PR TITLE
World population project

### PR DESCRIPTION
**Pull request by vansh-codes**
Closes: #480 

- **Title** : World Population Stats
- **Name:** : Vansh Chaurasiya
- **Idenitfy yourself:** : GSSOC'24 Contributor

### Describe the add-ons or changes you've made 📃
Updated the vanilla sj folder readme and added my project's link also.
Added files for my project along with a webp screenshot of the clone website.

### How Has This Been Tested? ⚙️

Tested locally

### Screenshots 📷

image
image
image
image
image

### Type of change ☑️

What sort of change have you made:

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Code style update (formatting, local variables)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] This change requires a documentation update

### Checklist: ☑️

- [x] My code follows the [Contributing Guidelines](https://github.com/Avdhesh-Varshney/WebMasterLog/blob/main/CONTRIBUTING.md) & [Code of Conduct](https://github.com/Avdhesh-Varshney/WebMasterLog/blob/main/CODE_OF_CONDUCT.md) of this project.
- [x] This PR does not contain plagiarized content.
- [x] I have performed a self-review of my own code.
- [x] I have commented my code, particularly wherever it was hard to understand.
- [x] My changes generate no new warnings.

### Note to reviewers 📄

Assign under GSSoC'24. 
Thank you!

### CODE REVIEW

**Comment by Avdhesh-Varshney** [comment]
resolve merge conflicts

**Comment by vansh-codes** [comment]
> resolve merge conflicts

Resolved ✅ 
Please check now @Avdhesh-Varshney

**Review by Avdhesh-Varshney — CHANGES_REQUESTED** [review]
add a readme file of project explanation

**Comment by vansh-codes** [comment]
> add a readme file of project explanation

Added ✅

File: Vanilla-JS-Projects/Basic/World-Population/README.md
Comment (Avdhesh-Varshney) [inline Vanilla-JS-Projects/Basic/World-Population/README.md:1]: image
properly write the readme file

**Pull request closed without merging**